README.rst

```diff
@@ -18,42 +17,40 @@ options, all from one common Sysl specification.
 
 The set of outputs is open-ended and allows for your own extensions. Sysl has been created with extensibility in mind and it will grow to support other representations over time.
 
-Installation
-------------
-If you are interested in trying out Sysl, you will need to install `Python 2.7 <https://www.python.org/downloads/>`_ .
+Prerequisites
+-------------
+If you are interested in trying out Sysl, you will need `Python 2.7 <https://www.python.org/downloads/>`_ .
 OSX users, we recommend installing Python 2.7 with `Homebrew <https://brew.sh/>`_  and adding it to the `PATH` in you `.bash_profile`::
 
   > brew install python
   > echo export PATH=/usr/local/opt/python/libexec/bin:\$PATH >> ~/.bash_profile
 
-You can then install Sysl with ::
-
-  > pip install sysl
+If you are behind a corporate proxy you might want to set up ``cntlm`` and a global ``pip.conf``
+file (see `official docs <https://pip.pypa.io/en/stable/user_guide/#config-file>`_ or `Stackoverflow <https://stackoverflow.com/a/46410817>`_).
 
-Now you can execute Sysl as command line tool with ::
+Installation
+------------
+Install Sysl with ::
 
-  > sysl    --root demo/petshop textpb -o out/petshop.txt /petshop
-  > reljam  --root demo/petshop model /petshop PetShopModel
+  > pip install sysl
 
-which is equivalent to ::
+Now you can execute Sysl as command line tools with ::
```

Comment:
s/as command line tool with/on the command line/

Reply:
Done.